packages/loader/container-loader/src/container.ts

```diff
@@ -1674,9 +1673,7 @@ export class Container
 		);
 
 		// The load context - given we seeded the quorum - will be great
-		await this.instantiateContextDetached(
-			false, // existing
-		);
+		await this.instantiateRuntime(codeDetails, undefined);
```

Comment:
Nit: the comment right above feels like it could use some updating.

Reply:
Weird one - I just removed it, not sure what it was originally trying to say :-P

Reply:
I imagine it meant to say "Then" instead of "The" but removing it seems fine too.